### DISCUSSION TITLE
Traduction française

### DISCUSSION THREAD

**Discussion by 5axes** [supermerill/SuperSlicer#1319] [General]
Plutôt que de poser les questions via des demandes sur le discord ou via les issues , j'ouvre une discussion sur la partie traduction en français du logiciel. L'objectif est de pouvoir partager des retours ou des validations  sur la traduction de SuperSlicer sur des termes qui sont parfois assez difficile à traduire simplement. et peut être sur des règles à respecter. 

Ma première question est faut il essayer de coller à une traduction PrusaSlicer qui parfois elle même porte à confusion mais pour une question de "compatibilité" ou usage partagé devrait être gardé ?

**Comment by 5axes** [comment]
Exemple de question ou de discussion ... le terme de "Solide" en Anglais que l'on trouve souvent traduit basiquement par Solide en Français mais qui a la signification de Plein / Pleine   Exemple Couche Solide : 
Dans Prusa Slicer 
![image](https://user-images.githubusercontent.com/11015345/122368071-4e730100-cf5d-11eb-8f6e-4cd2e0b52fd0.png)
Ma proposition de traduction 
![image](https://user-images.githubusercontent.com/11015345/122368185-65b1ee80-cf5d-11eb-9cfa-ba52c9a7d095.png)

On garde Solide ou on passe à Plein/Pleine ?

**Reply by tirasec** [reply]
+ plus 1 pour "Plein"

**Comment by 5axes** [comment]
Autre proposition  pour **Chronologie**  
![image](https://user-images.githubusercontent.com/11015345/122384858-64d48900-cf6c-11eb-97f6-85605c69379b.png)

par **Temps d'impression**

**Reply by supermerill** [reply]
J'aurais aussi pu mettre 'print time' en anglais.
J'ai mis chrologie car si l'on s'arrète sur un couche, on a sa 'date' d'impression, pas du tout son temps d'impression qui est 'layer time'

**Reply by 5axes** [reply]
Ce qui m'amène une autre question. **_Layer time_** : que j'ai traduit par **Temps de couche**   n'est pas à proprement parler le temps d'une couche (Au sens impression à un même Z) dans le monde Slic3r puisque qu'avec des supports le logiciel ( Idem sur Prusa) une couche est en réalité compté deux fois : Une fois pour l'impression et son remplissage , Une fois pour le niveau du support. Faut il garder Couche ou alors on est sur une Section ou autre vocable ?

![image](https://user-images.githubusercontent.com/11015345/122408773-e9ca9d00-cf82-11eb-892d-a2de4df7c6f6.png)

Ps A noter que ceci est un peu chiant avec l'option Renforcer sur le support sur les premières . le nombre de couche n'étant pas celui que l'on voit dans la simu.
![image](https://user-images.githubusercontent.com/11015345/122408960-0cf54c80-cf83-11eb-865e-476e2755a1a8.png)

**Reply by supermerill** [reply]
> A noter que ceci est un chiant avec l'option Renforcer sur le support sur les premières . le nombre de couche ...

pardon?

> Faut il garder Couche ou alors on est sur une Section ou autre vocable ?

On peut mettre `durée d'impression`, c'est peut-être mieux que `temps de couche`. Le fait que les supportsoit séparés, ce n'est pas si grave. idéalemrnt, il faudrait mettre cette info dans un tooltip ou la doc.

**Reply by 5axes** [reply]
Manquait : **Peu** à chiant

**Reply by supermerill** [reply]
Qu'est ce qui est un peu chiant? le fait qu'il n'y ai pas asser de place pour afficher sur une seule ligne?

**Reply by 5axes** [reply]
Non tu indiques un nombre de couche par exemple 100 mais en réalité si tu as un support ca va correspondre à 200 . Donc si tu parts d'une première impression , que tu te positionnes au niveau où tu voudrais voir arriver ton renfort et que tu note le numéro de couche dans le Rendu du G-code , Il faut bien penser selon les cas à diviser le nombre par 2

**Reply by supermerill** [reply]
ce problème mérite son ticket.

**Reply by 5axes** [reply]
Chez Qui ? C'est pareil chez Prusa .  Après je pense que c'est plus du truc à savoir donc comme il a été dit Tooltip ou Doc.

**Reply by supermerill** [reply]
Oui, en plus prusa modifie ses supports en ce moment.
Apres, il y a des tickets (issue) qui ne sont que chez prusa car les gens veulent que ce soit prusaslicer qui soit modifié ; il y en a qui sont ici et là-bas car les gens veulent une solution, peu importe le slicer, et il y en a qui ne sont qu'ici car les gens pensent que prusa n'ajoutera pas la fonction ou ils n'utilisent pas/plus prusaslicer.

**Comment by 5axes** [comment]
Le fameux Manifold  https://en.wikipedia.org/wiki/Manifold

Actuellement traduit par "Collecteur"  Au choix on garde un terme connu dans le domaine de la modélisation 3D .. Ou on le remplace par un Etanche ?

**Reply by supermerill** [reply]
+1 pour étanche, c'est mieux.

**Reply by tirasec** [reply]
Etanche, c'est mieux. Collecteur est une erreur de traduction de Deepl.com

**Comment by 5axes** [comment]
Nouvelle proposition : sur Coquillles 
![image](https://user-images.githubusercontent.com/11015345/122439846-4ab29f00-cf9c-11eb-88db-64e9a713313d.png)

A mon sens je pencherai plus pour **Modèles**

**Reply by supermerill** [reply]
Oui, ou géométries, originaux ..

**Comment by 5axes** [comment]
Et j'aurais besoin d'une explication sur le ToolTip du paramètre Compensation en XY **Intérieur**
![image](https://user-images.githubusercontent.com/11015345/122441568-f1e40600-cf9d-11eb-9a9d-25cbcc2530f0.png)

Il est écrit à l'origine : 
"The object will be grown/shrunk in the XY plane by the configured value (negative = inwards, positive = outwards). This might be useful for fine-tuning sizes.\n"
"This one only applies to the 'inner' shell of the object (!!! horizontal holes break the shell !!!)"

je comprend pas bien cette  alerte **(!!! horizontal holes break the shell !!!)**

**Reply by supermerill** [reply]
Ce n'est pas très bien dit.
Un tore n'a pas d'intérieur à proprement parler, pourtant ce paramètre va modifier la taille du "trou" du tore. 
En fait je ne sais pas trop si c'est nécessaire de garder cet avertissement.

**Reply by 5axes** [reply]
Je l'avais effectivement viré au départ.
Merci de ces précisions

**Reply by 5axes** [reply]
J'ai saisi l'idée qui était derrière ces 5 mots , mais effectivement pas simple à traduire .. ça reviendrait à dire je pense :
Attention la présence de trous  horizontaux peut transformer un profil fermé en profil ouvert et par la même rendre inopérant la correction sur ces profils etc etc

**Reply by supermerill** [reply]
ah oui, c'est ça. J'ai du écrire cela une fois où je m'était fait avoir et j'avais perdu à comprendre d'où venais le problème.

**Comment by 5axes** [comment]
Rectiligne ou Rectilinéaire ?

Dans PrusaSlicer le mode de remplissage suivant est dit **Rectiligne** 
![image](https://user-images.githubusercontent.com/11015345/122478564-f1616480-cfc9-11eb-919c-07fde9e23d3a.png)
Dans SuperSlicer la traduction donne actuellement **Rectilinéaire**   Mon choix irait plutôt sur la version PrusaSlicer

**Comment by supermerill** [comment]
Actuellement, `Height range modifiers` est traduit par l'innommable `la gamme de hauteur modificateurs de hauteur`
que je traduirais plutôt par
`modificateurs par plage de niveaux` (ou plage de hauteur?)
qu'en pensez-vous?

**Reply by 5axes** [reply]
J'aurais envie de **plage de hauteur** pour une certaine cohérence. 
![image](https://user-images.githubusercontent.com/11015345/123157272-9bc60580-d46a-11eb-9584-a93eb66ac2be.png)

Mais il faudrait aussi dans ce cas changer Gamme en **Plage**  

Et de même le texte Modificateur de la zone de hauteur ne devrait-il pas devenir  Modificateur de la **plage** de hauteur ?
![image](https://user-images.githubusercontent.com/11015345/123157538-f3647100-d46a-11eb-9eb2-5f8c6b00206b.png)

**Reply by 5axes** [reply]
Petite vérification sur PrusaSlicer c'est bien **Plage de Hauteur** ... Donc pour être raccord uniformisation souhaitée
![image](https://user-images.githubusercontent.com/11015345/123158472-1e9b9000-d46c-11eb-9906-03be551f04a9.png)
Par contre après c'est Zone
![image](https://user-images.githubusercontent.com/11015345/123158659-5efb0e00-d46c-11eb-96fb-fc5a10305068.png)

**Comment by supermerill** [comment]
extrudeur ou extrudeuse?

**Reply by 5axes** [reply]
Pas facile celui là .. mais bon pour être en phase avec le Larousse  : **Extrudeuse**

**Comment by 5axes** [comment]
Un petit qui n'est pas passé dans le Radar ... Ironing PP -> **Lissage**

& 

Milling -> **Fraisage**

![image](https://user-images.githubusercontent.com/11015345/123159037-d16bee00-d46c-11eb-82e5-ea76c3c30339.png)

Visiblement non présent dans Slic3r.po

**Reply by supermerill** [reply]
Corrigé

**Comment by supermerill** [comment]
`top fill` `top layer` `top fill`
=>`remplissage supérieur` ou `remplissage du dessus` ? Je penche pour le deuxième car il y a un problème de sens avec le premier

`bottom fill` `bottom layer`
=> `remplissage inférieur` ou `remplissage du dessous` ? pour être cohérent...

**Reply by 5axes** [reply]
Je plussois..; Mais il me semblait que c'était déjà le cas 

![image](https://user-images.githubusercontent.com/11015345/123162540-037f4f00-d471-11eb-8300-f13a3adf97ef.png)

Par contre peut être quelques traductions peut être à changer dans les infos bulles comme ci dessus avec  **remplissage du haut** ou **remplissage par le bas**

**Reply by supermerill** [reply]
`couches du dess(o)us pleines` ou `couches pleines du dess(o)us` ? je dirais plutôt le premier car c'est d'abord le fait qu'elle sont au-dessus ou dessous qui importe.

**Reply by 5axes** [reply]
Je sais pas moi je dirais  plutôt la 2  `Couches pleines du dess(o)us`   ca correspond à la lecture que l'on en a dans les paramètres 

![image](https://user-images.githubusercontent.com/11015345/123165119-19424380-d474-11eb-8e74-e6bdf4d078c4.png)

Couches pleines  (du)  Dessus
Couches pleines  (du)  Dessous

**Comment by 5axes** [comment]
J'ai une question et peut être une traduction à revoir : Coques d'interface 
![image](https://user-images.githubusercontent.com/11015345/123165937-20b61c80-d475-11eb-9a97-46e1ec2a207e.png)

Le nom est pas très explicit et je n'arrive même pas à comprendre ce que ce paramètre est censé faire ... J'ai joué avec sans jamais voir de différence .. Ca sert à quoi en fait ?

**Reply by supermerill** [reply]
aucune idée, jamais utilisé ni regardé.

edit: ca sert à ca 
![image](https://user-images.githubusercontent.com/6536403/123167935-8efbde80-d477-11eb-8f7a-d767c305cb69.png)

**Reply by 5axes** [reply]
Alors peut être : **Création d'interface entre coques** ?

![image](https://user-images.githubusercontent.com/11015345/123174161-fcac0880-d47f-11eb-9209-45c0b9e5f95b.png)

**Comment by 5axes** [comment]
Problème aussi sur traduction français !!!!! Pas d'accès aux boutons pour enlever la définition des couches manuelles :
En Anglais : 
![image](https://user-images.githubusercontent.com/11015345/123169326-7096e280-d479-11eb-87a3-429d460fcb96.png)

Sur la version Française

![image](https://user-images.githubusercontent.com/11015345/123169501-a89e2580-d479-11eb-9555-5490b21cc2f7.png)


Du coup il faut réduire les textes, proposition Comme sur PrusaSlicer  : 

Commencer à cette hauteur ->  **_Hauteur de début_**   ou encore plus court  **Hauteur début** 
Arrêter à cette hauteur -> **_Hauteur de fin_**    ou  **Hauteur fin**
Hauteur de couche -> **_Hauteur de couche_**    ou   **Hauteur couche**                                                   

![image](https://user-images.githubusercontent.com/11015345/123169810-0df21680-d47a-11eb-9da9-9282b3c00d6b.png)

**Reply by supermerill** [reply]
tu peux aussi mettre un `\n`, exemple: `Arrêter à\nla hauteur`
![image](https://user-images.githubusercontent.com/6536403/123171543-4135a500-d47c-11eb-82e6-13c89f494b81.png)

Après, hauteur de début / d'arrêt c'est très bien.

edit: épaisseur de couche ou hauteur? Je serais partant pour mettre `Epaisseur\ndes couches`

**Reply by 5axes** [reply]
oui mais Hauteur de début  c'est compatible PrusaSlicer :
![image](https://user-images.githubusercontent.com/11015345/123172426-6ecf1e00-d47d-11eb-923d-69271885725f.png)

**Reply by supermerill** [reply]
oui, c'est bien hauteur de début, j'aime.
Et la troisième boite, on fixe une épaisseur, pas une hauteur, non?
Enfin, ce n'est pas bien grave. Faudra que je pense à mettre "thickness" en anglais aussi.

**Reply by 5axes** [reply]
oui mais Hauteur de couche c'est correcte

**Reply by supermerill** [reply]
je laisse come ça alors, je lance une nightly avec tous les changements

**Comment by 5axes** [comment]
Pour **Jointure** ou **_Couture_** ?   les deux termes existent peut être une uniformisation à envisager

Perso je préfère le premier

**Comment by 5axes** [comment]
Tour de **_Nettoyage_**  ou Tour de **_Purge_** ?

**Reply by supermerill** [reply]
+1 pour purge

**Comment by 5axes** [comment]
Encore un : 
**mouvement brusque**  ou **saccade**       

Note : certains utilisent le terme de _Jerk_ mais dans le cas des machine utilisant Marlin ou d'autres systèmes nous ne sommes pas sur la vraie notion de Jerk qui est exprimé en mm/s3 mais ici c'est un notion de vitesse en mm/s

**Comment by 5axes** [comment]
**Annulations de filament**     -> je verrais plutôt **Rétraction du filament**
![image](https://user-images.githubusercontent.com/11015345/124483095-406d0f00-ddaa-11eb-98a0-932d54320b61.png)

**Comment by 5axes** [comment]
**Découper le rayon de fermeture de l'espacement**

Je verrais plus : 

**Distance de fermeture des faces disjointes**

![image](https://user-images.githubusercontent.com/11015345/124597375-b3868c00-de63-11eb-89c1-8315ab06f6a2.png)

Dans d'autres solution on parle de raccommodage (bof)

**Comment by 5axes** [comment]
Un peu long : 

**Faites fonctionner le ventilateur à la vitesse par défaut quand c'est possible** 

Un simple  :

**Ventilateur toujours actif** 


![image](https://user-images.githubusercontent.com/11015345/125209986-209f8480-e29d-11eb-8035-9b67314b2d2f.png)

**Comment by 5axes** [comment]
Vitesse de Dé rétraction

ou vitesse de Réinsertion ?
![image](https://user-images.githubusercontent.com/11015345/125210133-116d0680-e29e-11eb-96e1-56be51edcab7.png)

**Reply by supermerill** [reply]
Réinsertion sonne mieux.

**Comment by supermerill** [comment]
@5axes tu as changé ma traduction "brim"->"jupe" par "brim"->"bordure", pourquoi? je trouve que bordure c'est un "skirt" et jupe c'est un "brim", même si le nom anglais est complètement pourri, je trouve que c'est dommage de continuer a utiliser cette confusion en francais.

**Reply by 5axes** [reply]
C'est une bonne remarque qui mérite à mon avis discussion. Si je me rappel mes début en impression 3D avec Cura j'ai été aussi "perturbé" par la traduction  "brim"->"bordure" et "Skirt" -> "Jupe".  Parce que techniquement une Jupe c'est plus un élément qui part d'un élément comme la jupe par exemple sur les voitures F1 dans les années 70 .

![1200px-Lotus_Modell_79_under-body svg](https://user-images.githubusercontent.com/11015345/128029575-08bcf72b-091d-464b-8044-215fbeb7818e.png)

Et bordure .. ben c'est une bordure comme la bordure qui entoure un parterre de fleur.  
Donc je suis en phase avec toi pour moi **Brim** et **Skirt** employé sur les Slicers ne sont pas correcte. Mais maintenant si l'on reprend le vocabulaire employé et compris par les utilisateurs de solutions 3D  par Exemple si l'on prend la traduction de PrusaSlicer, Cura  ou IdeaMaker une bordure et une jupe c'est ça :

![image](https://user-images.githubusercontent.com/11015345/128044867-6578bbb7-fc1d-40ff-8e50-50a67ade1060.png)

Donc mon point de vue était le suivant : **Oui je suis d'accord avec toi la traduction n'est pas bonne** ( ou plus exactement ce n'est pas la traduction c'est au départ  **Brim** et **Skirt** qui sont mal employé ).  Maintenant considérant la traduction SuperSlicer  je pense que pour faciliter la compréhension et l'utilisation du logiciel par des utilisateurs qui viennent pour la plus part ( si ce n'est tous) d'autres solutions,  il était plus simple de garder "cette erreur" est d'accepter que Brim et Skirt soit mal utilisé pour faciliter la compréhension.   

**Autre considération** si tu gardes la traduction "correcte" au final c'est toi le patron. Il serait alors bon de corriger partout ou cette traduction est adopté ( c'est l'autre raison qui m'a poussé à revenir sur le consensus Prusa / Cura et peut être d'autres ? )   Car dans les traductions des débits par exemple on garde Jupe par exemple dans ta traduction pour Skirt ( skirt_extrusion_width). Je n'ai plus la liste en tête mais il y a plusieurs paramètres qui du coup n'étaient pas traduit correctement dans ce cas.

![image](https://user-images.githubusercontent.com/11015345/128031575-3c67be7c-1751-4701-bc85-5383e37ce301.png)

**Reply by supermerill** [reply]
Je vais demander à mon oncle qui pipe pas un mot d'anglais et utilise cura, pour voir si il se souvient de quel terme correspond à quoi. Si il se "trompe", je garde ma version ;p

> Il serait alors bon de corriger partout ou cette traduction est adopté 

Oui, dans la version actuelle, il y a un mélange constant, j'imagine dû aux différentes corrections et traduction faites par nous à chaque fois, d'où la concertation nécessaire.

**Reply by 5axes** [reply]
A mon avis les échanges viennent à l'origine du mix de départ avec PrusaSlicer ?

**Reply by supermerill** [reply]
possible, j'avais pourtant fait gaffe à vérifier tout les endroits avec 'skirt' et 'brim'

**Reply by supermerill** [reply]
tous ceux à qui j'ai demandé utilisent instinctivement "jupe" pour désigner le "brim".

A la limite, on peux changer les mots pour éviter les confusions. Quelque chose comme "attache" et "frontière"

**Reply by 5axes** [reply]
On va monter un comité AFNOR pour la normalisation des termes  de fabadd français :)

**Comment by 5axes** [comment]
Multimatériaux ou Multi-matériaux ?

**Reply by supermerill** [reply]
Comme tu veux.
moultipass

**Comment by 5axes** [comment]
Bon j'ai un peu de mal à suivre .. Donc avec 2.3.57.0 on est revenu comme avant  la jupe est une bordure et la bordure une jupe ... C'est quoi au final le choix définitif  ?
![image](https://user-images.githubusercontent.com/11015345/132327604-ef2bfd42-3028-497a-9e73-42c48a7292a5.png)

**Comment by 5axes** [comment]
Message d'erreur associé au paramètre [Hauteur maximale d'impression](https://github.com/5axes/SuperSlicer-FRDocumentation/blob/main/src/variable/max_print_height.md) non traduit

![image](https://user-images.githubusercontent.com/11015345/140072263-d7125a09-b3e4-4300-8f5a-3fec3caaa7dc.png)

**Reply by supermerill** [reply]
Corrigé pour la prochaine version.

**Reply by 5axes** [reply]
Du coup on peut aussi rajouter celui là : 
![image](https://user-images.githubusercontent.com/11015345/140329068-a273c05e-45ea-4677-ae72-4871f07ecd8c.png)

L'objet de départ est bien dans la zone, mais pas le parcours calculé ( peut être le cas des supports ou de la jupe / bordure)

**Reply by supermerill** [reply]
j'ai corrigé ceux-la

```
    case ObjectOutside: text = _u8L("An object outside the print area was detected."); break;
    case ToolpathOutside: text = _u8L("A toolpath outside the print area was detected."); error = true; break;
    case SlaSupportsOutside: text = _u8L("SLA supports outside the print area were detected."); error = true; break;
    case SomethingNotShown: text = _u8L("Some objects are not visible."); break;
    case ObjectClashed:
        text = _u8L( "An object outside the print area was detected.\n"
                  "Resolve the current problem to continue slicing.");
        error = true;
```

**Comment by 5axes** [comment]
Comment traduire GapFill  / Gap Fill?

- Remplissage des vides
- Remplissage des espaces
- Remplissage des interstices

- Bouchage des espaces     ( Remplissage état déjà le terme pour le remplissage interne, ou pour le remplissage des ponts ...)
- Bouchage des interstices

**Comment by 5axes** [comment]
Fuzzy Skin   -> Surfaces Irrégulières 
sur d'autres solution on a la traduction **Surfaces Floues** ou **Surfaces Rugueuses**

**Comment by 5axes** [comment]
Vue non traduit 
![image](https://user-images.githubusercontent.com/11015345/186162925-f66df2d4-df1f-47b9-82d4-49dd5f4b33ba.png)
et sur la même fenêtre : 
![image](https://user-images.githubusercontent.com/11015345/186163074-5d525740-e8e6-48da-a0de-8859ff51f4a0.png)

**Comment by 5axes** [comment]
Espacement de la Jupe -> Paramètre avec jupe dans la section Bordure .
![image](https://user-images.githubusercontent.com/11015345/186705280-5d611b07-9612-4cf8-81a5-0433cad8403d.png)

Proposé : **Espacement de la bordure**
